tests/test_outcome.py

```diff
@@ -0,0 +1,25 @@
+from roulette import Outcome
+
+from unittest import TestCase
+
+
+class TestOutcome(TestCase):
+    def setUp(self):
+        self.oc1 = Outcome("Red", 1)
+        self.oc2 = Outcome("Red", 1)
+        self.oc3 = Outcome("Black", 2)
+
+    def test_compare_objects_with_identical_names(self):
```

Comment:
test name should reflect what is being tested. a better name can be `test_objects_with_same_name_are_equal`